### PR TITLE
Release cardano-api-8.45.0.0

### PR DESCRIPTION
**Pull request by smelc**
# Changelog

```yaml
- description: |
    Release cardano-api-8.45.0.0
# uncomment types applicable to the change:
  type:
  # - feature        # introduces a new feature
  # - breaking       # the API has changed in a breaking way
  # - compatible     # the API has changed but is non-breaking
  # - optimisation   # measurable performance improvements
  # - improvement    # QoL changes e.g. refactoring
  # - bugfix         # fixes a defect
  # - test           # fixes/modifies tests
  # - maintenance    # not directly related to the code
  - release        # related to a new release preparation
  # - documentation  # change in code docs, haddocks...
```

# Context

CHaP PR: https://github.com/IntersectMBO/cardano-haskell-packages/pull/742

### CODE REVIEW

File: cardano-api/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+
+- Implement estimateBalancedTxBody and estimateOrCalculateBalancedTxBody
+  (feature, compatible)
+  [PR 511](https://github.com/IntersectMBO/cardano-api/pull/511)
+
+- Add support for CC keys in the SomeAddressVerificationKey type
+  (feature, breaking)
+  [PR 514](https://github.com/IntersectMBO/cardano-api/pull/514)
+
+- - Add `getReferenceInputsSizeForTxIds` function.
```
Comment (carbolymer) [inline cardano-api/CHANGELOG.md:13]: This shouldn't be in changelog as it was released previously. Is it a bug in the changelog script?

**Review by carbolymer — CHANGES_REQUESTED** [review]
Redundant entry in changelog

File: cardano-api/CHANGELOG.md (hunk 1 above)
Comment (smelc) [reply cardano-api/CHANGELOG.md:13]: Good catch, this probably comes from this interleaving: 

image

Comment (carbolymer) [reply cardano-api/CHANGELOG.md:13]: Linearized view of GH commits is ass.
image
Yeah. Seems that the script looked at `428fa68e4` or `296a61656` but they aren't children of 8.44 tag `530d44384` so they shouldn't be taken into account.

Comment (smelc) [reply cardano-api/CHANGELOG.md:13]: This is caused by `git log`'s behavior:

```
       List commits that are reachable by following the parent links from the given commit(s), but exclude commits
       that are reachable from the one(s) given with a ^ in front of them.

       A special notation "<commit1>..<commit2>" can be used as a short-hand for "^<commit1> <commit2>".
```

So in this case the script executes `git log --merges --oneline "cardano-api-8.44.0.0..HEAD"` i.e.:

```
git log --merges --oneline ^cardano-api-8.44.0.0 HEAD
```

And indeed, `428fa68e4` is a parent of `HEAD` that is not a parent of `cardano-api-8.44.0.0`, so it gets listed. What we want is the list of commits between the two commits we give, not git log's behavior of using the parent link.

Comment (smelc) [reply cardano-api/CHANGELOG.md:13]: I believe we need to add [--ancestry-path](https://git-scm.com/docs/git-log#Documentation/git-log.txt---ancestry-pathltcommitgt) to our `git log` invokation.

**Comment by smelc** [comment]
PR updating API in CLI: https://github.com/IntersectMBO/cardano-cli/pull/720

**Pull request merged**